### PR TITLE
Adding example for Calling external file in terraform

### PR DESCRIPTION
**Pull request by dkuldeep22**

### CODE REVIEW

**Review by yogeshprasad — APPROVED** [review]

File: docs/resources/dashboard_json.md [hunk 1]
```diff
@@ -115,6 +115,142 @@ EOF
 }
 ```
 
+## Reading from external file
```
Comment (Margarita-Staneva) [inline docs/resources/dashboard_json.md:118]: Reading from an External File

File: docs/resources/dashboard_json.md [hunk 2]
```diff
@@ -115,6 +115,142 @@ EOF
...
+Following is the example dashboard with sections and parameters from external file.
```
Comment (Margarita-Staneva) [inline docs/resources/dashboard_json.md:119]: Below is the example dashboard with sections and parameters from an external file.

File: docs/resources/dashboard_json.md [hunk 3]
```diff
@@ -115,6 +115,142 @@ EOF
...
+                                  {
+                                    section1   = file("<path>/section1.json"),
+                                    section2   = file("<path>/section2.json"),
+                                    parameters = file("<path>/params.json")
+                                  }
+                                )
+}
+```
+
+Following are the sample files.
```
Comment (Margarita-Staneva) [inline docs/resources/dashboard_json.md:134]: The sample files are listed below.

File: docs/resources/dashboard_json.md [hunk 4]
```diff
@@ -115,6 +115,142 @@ EOF
...
+
+* dashboard_file.txt
+```hcl
+{
+  "name": "Terraform Test Dashboard Json",
```
Comment (Margarita-Staneva) [inline docs/resources/dashboard_json.md:139]: Terraform Test Dashboard JSON

File: docs/resources/dashboard_json.md [hunk 5]
```diff
@@ -115,6 +115,142 @@ EOF
...
+  "selectedLabel": "Label",
+  "value": "test"
+  }
+}
+```
+
+
+
+
+* Note - If there are dynamic variables present in Wavefront parameters, then parameters needs to be present in separate file.
```
Comment (Margarita-Staneva) [inline docs/resources/dashboard_json.md:252]: I think the note shouldn't be a bullet. I'd rather put it on a separate line like that:

**Note**: If there are dynamic variables in the Wavefront parameters, then these parameters must be present in a separate file.

**Review by Margarita-Staneva — COMMENTED** [review]
I have some comments, please take a look.

File: docs/resources/dashboard_json.md (hunk 1 above)
Comment (dkuldeep22) [reply docs/resources/dashboard_json.md]: Done. Thank you!

**Review by Margarita-Staneva — APPROVED** [review]
LGTM

**Pull request merged**